R/correctionsataglance-data.R

```diff
@@ -117,12 +117,23 @@ formatDateRange <- function(startD, endD){
 }
 
 formatDataList <- function(dataIn, type, ...){
+  
   args <- list(...)
   
   if(length(dataIn) == 0){
     return()
   }
   
+  if(type == 'APPROVALS' && length(dataIn)>0){
```

Comment:
If keeping this as a single method, moving all of the APPROVALS stuff into a single if block would make this more readable.  Big picture:

if (Approvals) {
//do this
} else if (Meta) {
//do that
} else {
//do we expect this case to happen
}
//do stuff common to all list formatting